packages/test/src/test-workflows.ts

```diff
@@ -363,6 +363,40 @@ function makeSetPatchMarker(myPatchId: string, deprecated: boolean): coresdk.wor
   };
 }
 
+function makeUpdateActivationJob(
+  id: string,
+  protocolInstanceId: string,
+  name: string,
+  input: unknown
+): coresdk.workflow_activation.IWorkflowActivationJob {
+  return {
+    doUpdate: {
+      id,
+      protocolInstanceId,
+      name,
+      input: toPayloads(defaultPayloadConverter, input),
```

Comment:
Shouldn't the second argument of `toPayloads` be an array? If so, the `input` argument should be `unknown[]`.

Reply:
Yes, but `input` is a rest parameter in `toPayloads`, its a variadic function. I've changed it anyways for consistency.